<!--In order for the trajectory follower to work, it'll have to wait to read the full offsetted trajectory-->
<!--We'll need to publish it to a new topic and have the -->
<launch>
    <!--Safety controller-->
    <!-- <node pkg="wall_follower" name="wall_follower" exec="safety_controller" output="screen">
        <param from="$(find-pkg-share wall_follower)/params.yaml"/>
    </node>

    <node pkg="path_planning" exec="trajectory_loader" name="trajectory_loader">
        <param name="trajectory" value="$(find-pkg-share luigi_mansion)/lanes/full-lane.traj"/>
    </node>

    <node pkg="localization" exec="particle_filter" name="particle_filter">
        <param from="$(find-pkg-share path_planning)/config/sim/pf_config.yaml"/>
    </node>

    <node pkg="luigi_mansion" exec="city_drive" name="city_drive">
    </node>

    <node pkg="luigi_mansion" exec="basement_point_pub" name="BasementPointPub">
    </node>  -->

    <node pkg="luigi_mansion" exec="stop_detector" name="stop_detector" output="screen">
    </node>

    <node pkg="luigi_mansion" exec="homography_transformer" name="homography_transformer">
    </node>
</launch>
